src/roll20.pyj

```diff
@@ -215,6 +215,10 @@ def template(request, name, properties):
 
     return result
 
+def rollAvatarDisplay(request):
+    return "[x](" + request.character.avatar + "#.png)"
+    #postChatMessage("[x](" + request.character.avatar + "#.png)", request.character.name)

```

Comment:
oh, that's smart. You're overriding Roll20's image detection in case the URL doesn't have the right extension, but adding it as a query string.
I'd love to have seen that as a comment to clarify for future maintenance needs.

Reply:
Just noticed, the `postChatMessage` is commented out :p